app/models/redis_notification.rb

```diff
@@ -5,6 +5,11 @@ class RedisNotification < ActiveRecord::Base
 
   belongs_to :subject, polymorphic: true
   belongs_to :current_user, optional: true, class_name: "User"
+  belongs_to :project, optional: true
+  belongs_to :issue, optional: true
+  belongs_to :user, optional: true
+  belongs_to :member, optional: true
+  belongs_to :role, optional: true
```

Comment:
Hm, I'm not sure here. This feels a bit too inflexible to use database based-associations here IMO. 🤔 

What happens if we need to make further additions or want to track an even more nested model. Instead I'm thinking about using `serialize` with a JSON coder to save any data attribute we want to pass to Redis. With using serialize here, we'd make sure that it's extendable in the furture and as well still make it Database driver independent, as rails would handle the serialization and deserialization for us.